src/status.rs

```diff
@@ -298,19 +298,32 @@ macro_rules! status_codes {
     (
         $(
             $(#[$docs:meta])*
-            ($num:expr, $konst:ident, $phrase:expr);
+            ($num:expr, $name:ident, $phrase:literal $(, $alias:ident)*);
         )+
     ) => {
         impl StatusCode {
+            const fn from_u16_or_panic(num: u16) -> Self {
+                if num < 100 || 999 < num {
+                    panic!()
+                }
+                Self(unsafe { NonZeroU16::new_unchecked(num) })
+            }
+
         $(
             $(#[$docs])*
-            pub const $konst: StatusCode = StatusCode(unsafe { NonZeroU16::new_unchecked($num) });
+            pub const $name: StatusCode = StatusCode::from_u16_or_panic($num);
+            $(
+                #[doc = concat!("Alias of [`", stringify!($name),
+                                "`](`Self::", stringify!($name), "`).")]
+                pub const $alias: StatusCode = StatusCode::$name;
```

Comment:
Should these be deprecated now, and removed for http 1.0?

Reply:
I dunno.  Status code phrases does really matter and deprecating those names just seems like an unnecessary burden for users.  It’s not like keeping those names is considerable maintenance burden.

Reply:
Fair point, thanks for entertaining my random musings :)